utils/getInvoices/tests/getInvoices.inflicion.test.ts

```diff
@@ -6,144 +6,549 @@ import { ServiceType } from '@utils/constants'
+        expect.objectContaining({ type: ServiceType.Inflicion})
+      )
     })
   })
 
-  it('should NOT load Inflicion from company without inflicion', () => {
-    const company: Partial<IRealestate> = {
-      inflicion: false,
-    }
-    const service: Partial<IService> = {
-      inflicionPrice: 101,
-    }
-    const payment: Partial<IPayment> = null
-    const prevPayment: Partial<IPayment> = null
-
-    const invoices = getInvoices({
-      company,
-      service,
-      payment,
-      prevPayment,
-    })
-
-    expect(invoices).not.toContainEqual(
-      expect.objectContaining({ type: ServiceType.Inflicion })
-    )
+  describe('props: { payment }', () => {
+    it('should NOT load when payment = null', () => {
+      const payment: Partial<IPayment> = null
+
+      const invoices = getInvoices({
+        payment,
+      })
+
+      expect(invoices).not.toContainEqual(
+        expect.objectContaining({ type: ServiceType.Inflicion})
+      )
+    })
+    it('should NOT load when payment = { invoice: null }', () => {
+      const payment: Partial<IPayment> = {
+        invoice: null,
+      }
+
+      const invoices = getInvoices({
+        payment,
+      })
+
+      expect(invoices).not.toContainEqual(
+        expect.objectContaining({ type: ServiceType.Inflicion})
+      )
+    })
+    it('should NOT load when payment = { invoice: [] }', () => {
+      const payment: Partial<IPayment> = {
+        invoice: [],
+      }
+
+      const invoices = getInvoices({
+        payment,
+      })
+
+      expect(invoices).not.toContainEqual(
+        expect.objectContaining({ type: ServiceType.Inflicion})
+      )
+    })
+    it('should NOT load when payment = { invoice: [Cleaning] }', () => {
+      const payment: Partial<IPayment> = {
+        invoice: [
+          {
+            type: ServiceType.Cleaning,
+            price: 10,
+            sum: 100,
+          },
+        ],
+      }
+
+      const invoices = getInvoices({
+        payment,
+      })
+
+      expect(invoices).not.toContainEqual(
+        expect.objectContaining({ type: ServiceType.Inflicion})
+      )
+    })
+    it('should load when payment = { invoice: [Inflicion] }', () => {
+      const payment: Partial<IPayment> = {
+        invoice: [
+          {
+            type: ServiceType.Inflicion,
+            price: 10,
+            sum: 110,
+          },
+        ],
+      }
+
+      const invoices = getInvoices({
+        payment,
+      })
+
+      expect(invoices).toContainEqual({
+        type: ServiceType.Inflicion,
+        price: 110,
+        sum: 110,
+      })
+    })
   })
 
-  it('should NOT load Inflicion from payment without inflicion', () => {
-    const company: Partial<IRealestate> = null
-    const service: Partial<IService> = null
-    const payment: Partial<IPayment> = {
-      invoice: [],
-    }
-    const prevPayment: Partial<IPayment> = null
+  describe('props: { service, company, prevPayment }', () => {
```

Comment:
Add missing `props: { service, company }`, `props: { service, company, prevService }` and `props: { service, company, prevService, prevPayment }` blocks, because inflicion can be loaded in some scenarious